### ISSUE TITLE
Use RunKit to check JS library versions

### ISSUE THREAD

**Issue by rmzelle** [citation-style-language/csl-validator#12]
As a personal reminder, I have a little bit of RunKit code that shows the latest libraries in https://cdnjs.com/, which makes it easier to spot which of the JS libraries in the CSL validator, formatter, and my ref-extractor are out-of-date:

https://runkit.com/rmzelle/59e0e574834ff30012743623

**Comment by zuphilip** [comment]
FYI We use https://greenkeeper.io/ in our project https://github.com/locdb/locdb-frend which will then automatically create PR and issues to keep the libraries up-to-date. But the architecture is different then here and you need a CI first.

**Comment by rmzelle** [comment]
Yes, I briefly looked at a few of those CI services to keep libraries up-to-date, but they don't ever seem to cover referenced libraries in HTML files. More for package managers like NPM.

This snippet just saves me from having to check each library one by one on CDNJS.
